_sass/syntax-highlighting.scss

```diff
@@ -0,0 +1,82 @@
+// Syntax highlighting styles
+.highlight {
+    background: #ddd;
+    padding: 10px;
+
+    .c { // Comment
+        color: #998; 
+        font-style: italic;
+    }
+    .err { // Error
+        color: #a61717;
+        background-color: #e3d2d2;
+    }
+    .k { // Keyword
+        font-weight: bold;
+    }
+    .o { // Operator
+        font-weight: bold;
+    }
+    .cm { // Multiline comment
+        color: #998;
+        font-style: italic;
+    }
+    .cp {
+        color: #999;
+        font-weight: bold;
+    }
+    .c1    { color: #998; font-style: italic } // Comment.Single
```

Comment:
Might want to continue going on with the style change. Perhaps even remove some of them.